docs/content/Infrastructure.md

```diff
@@ -144,7 +149,7 @@ Shared modules between android-test and gradle.
 1. Make sure new project version specified in develop head
 1. Manually run [Teamcity configuration (internal)](http://links.k.avito.ru/releaseAvitoTools)
 1. Use new version in `avito`
-1. Create PR with new `infraVersion`
+1. Create PR with new `infraVersion` and new `projectVersion`
```

Comment:
more like `Create PR with infraVersion= released version` `and bump projectVersion`